ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/apps/page.tsx

```diff
@@ -38,6 +40,34 @@ export default function Page() {
         icon={<Squares2X2Icon className="h-5 w-5 text-white" />}
         links={navLinks}
         title="Apps"
+        action={
+          <HoverCardRoot>
+            <HoverCardTrigger asChild>
+              {/* To do: wire this to the docs */}
```

Comment:
```suggestion
              {/* TODO: wire this to the docs */}
```

Nit: using [conventional comments](https://conventionalcomments.org) makes todos more discoverable. If we always use "TODO" then we only need to search "TODO" to find them all. Using a mix of "TODO" and "To do" makes it easier to lose track